mpf/tests/machine_files/high_score/modes/high_score/config/high_score.yaml

```diff
@@ -19,3 +19,4 @@ high_score:
       - MPF: 23
     loops:
       - JK: 42
+  reverse_sort: False
```

Comment:
Maybe this is from me misunderstanding how the system works in a real life example, but the reason I've put this is here is that it needs to always be `False` when running the tests, as having a value of `True` breaks multiple other tests.

If a user has built their own config for their machine, sets `reverse_sort: True`, and for some reason or another needs to run tests, would the value in the default config still be used for the test, or would it use the user's config?

Reply:
My concern was less about users who set `reverse_sort: True`, which should work according to the changes you've made. I'm more curious about users who don't have _anything_ set for `reverse_sort`—which right now, is everybody. 

Since you've set the default value to be `False`, then in theory all of the existing tests should pass with no `reverse_sort` setting at all. This is important, because we don't want to force everybody to add `reverse_sort`. 

If you remove this line, will all the tests still pass? If so, that's great and we should keep it that way 😃 If not, then the default value isn't being used and it'll need a fix before we merge.